gamification/process/update.py

```diff
@@ -135,6 +136,33 @@ def get_participant_objects():
     return participants
 
 
+def get_meta_review_participant_objects():
+    return meta_review_participants.objects.all()
+
+
+def update_participants_data_with_meta_review(participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the meta-review is completed then it calls
+    'Completed Meta-Review' activity.
+    """
+    pos_in_meta_review = participant.pos_in
+    pos_out_meta_review = participant.pos_out
+    neg_in_meta_review = participant.neg_in
+    neg_out_meta_review = participant.neg_out
+    if pos_in_meta_review > 0 or pos_out_meta_review > 0 or \
+       neg_in_meta_review > 0 or neg_out_meta_review > 0:
+        """
+        Add a meta-review activity to the participant
+        """
+        activity = Activity.object.create(
+            name=participant,
+            activity='Completed Meta-Review')
+        participant.activities.add(activity)
```

Comment:
Now, this participant is a meta-review object.
You have to get a gamification participant first.
```
from gamification.models import Participant
gam_participant = Participant.objects.get_or_create(username=participant.login)
```